types/scte224v20200407/sctestructures.go

```diff
@@ -42,6 +46,46 @@ type Media struct {
 	MediaPoints []*MediaPoint `xml:"http://www.scte.org/schemas/224 MediaPoint" json:"mediaPoints,omitempty"`
 }
 
+func (m *Media) Get2018() (*scte224_2018.Media, error) {
+	encodedMedia, err := xml.Marshal(m)
+	if err != nil {
+		return nil, fmt.Errorf("error attempting to downgrade the 2020 element, %+v", err)
+	}
+
+	// Unmarshal into 2018 struct
+	var media2018 *scte224_2018.Media
+	err = xml.Unmarshal(encodedMedia, &media2018)
```

Comment:
how does this benchmark?  I think I avoided a marshal-unmarshal route with 2015-2018 conversions both due to the namespace difference as well as thinking marshalling performance would be terrible.  Might not be a big deal though mostly curious.

Reply:
Valid concern with Unmarshaling/marshaling. I am skeptical about it too. 

Yet to benchmark, was going to do so while adding unit tests.

We could leave the PR open, until I benchmark and observe results.

Reply:
We still have to dereference each of the values, if we were to manually copy from one struct to another. 

So I thought, at least theoretically, there wouldn’t be too much difference between marshaling/unmarshaling approach and copying approach.